### ISSUE TITLE
foundation-cli@beta - unable to create project from template

### ISSUE THREAD

**Issue by detectedstealth** [zurb/foundation-cli#14]
```
$ foundation new --framework sites --template advanced
? What's the project called? warply

            .
           /|     ,
      , /|/  \/| /|       Thanks for using ZURB Foundation for Sites!
     /|/       |/ |       -------------------------------------------
 |___|            |___|   Let's set up a new project.
 \___|  ^^   ^^   |___/   It shouldn't take more than a minute.
     | -[O]--[O]- |
     |    ___,    |
     |    ...     |
      \__________/


```Downloading the project template.../usr/local/lib/node_modules/foundation-cli/node_modules/update-notifier/node_modules/configstore/node_modules/graceful-fs/polyfills.js:14
  chdir.call(process, d)
        ^

Error: ENOENT: no such file or directory, uv_chdir
    at Error (native)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/update-notifier/node_modules/configstore/node_modules/graceful-fs/polyfills.js:14:9)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/bower/node_modules/bower-config/node_modules/graceful-fs/polyfills.js:18:9)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/bower/node_modules/graceful-fs/polyfills.js:14:9)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/npm/node_modules/graceful-fs/polyfills.js:18:9)
    at /usr/local/lib/node_modules/foundation-cli/lib/commands/new.js:84:15
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/foundation-cli/node_modules/exec/exec.js:90:5)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:818:16)
```

**Comment by gakimball** [comment]
Just deployed a new version that fixes this. Run `npm install foundation-cli@beta --global` to get it.

**Comment by detectedstealth** [comment]
The new deployment doesn't seem to have fixed the issue:

```
$ npm uninstall foundation-cli --global
unbuild foundation-cli@2.0.0-beta.3
$ npm install foundation-cli@beta --global
npm WARN deprecated exec@0.1.4: deprecated in favor of builtin child_process.execFile
/usr/local/bin/foundation -> /usr/local/lib/node_modules/foundation-cli/bin/cli.js

> foundation-cli@2.0.0-beta.3 postinstall /usr/local/lib/node_modules/foundation-cli
> node lib/welcome.js

foundation-cli@2.0.0-beta.3 /usr/local/lib/node_modules/foundation-cli
├── is-root@1.0.0
├── open@0.0.5
├── async@0.9.2
├── exec@0.1.4
├── colors@1.1.2
├── semver@4.3.6
├── nopt@3.0.4 (abbrev@1.0.7)
├── string-length@1.0.1 (strip-ansi@3.0.0)
├── multiline@1.0.2 (strip-indent@1.0.1)
├── which@1.2.0 (is-absolute@0.1.7)
├── paint-by-number@1.0.0 (chalk@1.1.1)
├── rimraf@2.4.3 (glob@5.0.15)
├── inquirer@0.8.5 (ansi-regex@1.1.1, figures@1.4.0, cli-width@1.1.0, through@2.3.8, readline2@0.1.1, chalk@1.1.1, lodash@3.10.1, rx@2.5.3)
├── update-notifier@0.2.2 (is-npm@1.0.0, semver-diff@2.0.0, chalk@0.5.1, latest-version@1.0.1, configstore@0.3.2)
├── npm@2.14.10
└── bower@1.6.5
$ foundation new --framework sites --template advanced
? What's the project called? warply

            .
           /|     ,
      , /|/  \/| /|       Thanks for using ZURB Foundation for Sites!
     /|/       |/ |       -------------------------------------------
 |___|            |___|   Let's set up a new project.
 \___|  ^^   ^^   |___/   It shouldn't take more than a minute.
     | -[O]--[O]- |
     |    ___,    |
     |    ...     |
      \__________/


Downloading the project template.../usr/local/lib/node_modules/foundation-cli/node_modules/update-notifier/node_modules/configstore/node_modules/graceful-fs/polyfills.js:14
  chdir.call(process, d)
        ^

Error: ENOENT: no such file or directory, uv_chdir
    at Error (native)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/update-notifier/node_modules/configstore/node_modules/graceful-fs/polyfills.js:14:9)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/bower/node_modules/bower-config/node_modules/graceful-fs/polyfills.js:18:9)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/bower/node_modules/graceful-fs/polyfills.js:14:9)
    at process.chdir (/usr/local/lib/node_modules/foundation-cli/node_modules/npm/node_modules/graceful-fs/polyfills.js:18:9)
    at /usr/local/lib/node_modules/foundation-cli/lib/commands/new.js:85:15
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/foundation-cli/node_modules/exec/exec.js:90:5)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:818:16)
```

Unless I am missing a step?

**Comment by gakimball** [comment]
Hmmm. If you just type `foundation new` and go through the steps manually, it will work. I'll look into it further.

**Comment by detectedstealth** [comment]
Yeah you are right using foundation new works as expected. Thanks.

**Comment by gakimball** [comment]
Alright, fixed via c5b12c02e725a18130ebe29f667d45de5308f4e9

And I deployed a new version.
